packages/editor/src/components/rich-text/README.md

```diff
@@ -57,6 +57,25 @@ Render a rich [`contenteditable` input](https://developer.mozilla.org/en-US/docs
 
 *Optional.* A list of autocompleters to use instead of the default.
 
+### `annotations: Array`
```

Comment:
Why is this in the docs, if it's RichText provides this by itself.?

Reply:
I've added prop support for annotations to `RichText` in https://github.com/WordPress/gutenberg/commit/0371662688bbb8a90af3663b25ba3c44aaa4c473 because what a block might want to do with text annotations could not be covered by a simple identifier.

I agree that before the documentation was redundant.